nixpkgs_review/cli/__init__.py

```diff
@@ -223,6 +223,13 @@ def common_flags() -> List[CommonFlag]:
             default=read_github_token(),
             help="Github access token (optional if request limit exceeds)",
         ),
+        CommonFlag(
+            "--nix-flavor",
```

Comment:
nom is not a nix flavor but a wrapper around nix itself.

Reply:
I struggled to find a good name for this, do you have any suggestions?

Reply:
I think `--build-graph` is a good description